src/classes/OPP_PrimaryContact_TEST.cls

```diff
@@ -34,137 +34,200 @@
 * @description Test class for the Opportunity Primary Contact lookup.
 */
 @isTest
-public with sharing class OPP_PrimaryContact_TEST {
+private with sharing class OPP_PrimaryContact_TEST {
+    private static final String CLOSE_WON_STAGE = UTIL_UnitTestData_TEST.getClosedWonStage();
 
     /*******************************************************************************************************
     * @description Tests insert, update, and delete of the primary opportunity contact role based on the opportunity
     * Primary Contact field.
     ********************************************************************************************************/
-    static testMethod void testCreateUpdateOCR() {
-        list<Contact> conList = UTIL_UnitTestData_TEST.CreateMultipleTestContacts(2);
-        insert conList;
+    @isTest
+    private static void shouldUpdateCreateOrDeleteOCRsBasedPrimaryContactFieldOnOppChange() {
+        List<Contact> contacts = UTIL_UnitTestData_TEST.CreateMultipleTestContacts(2);
+        insert contacts;
 
         Opportunity opp = new Opportunity(
             Name = 'foo',
-            Primary_Contact__c = conList[0].id,
-            CloseDate = system.today(),
-            StageName = UTIL_UnitTestData_TEST.getClosedWonStage(),
+            Primary_Contact__c = contacts[0].Id,
```

Comment:
Even though this is legacy code, rather than have all of these create new Opps with a name of "foo", what would you think about it naming the Opp with the name of the test method, this way the name is unique?